test/parser/testlogfileparser.py

```diff
@@ -17,6 +17,7 @@
 
 import cclib
 import pytest
+from _pytest.monkeypatch import MonkeyPatch
```

Comment:
Does
```suggestion
from pytest.monkeypatch import MonkeyPatch
```
not work?  `_pytest` is the private implementation that we technically shouldn't rely on.

Reply:
Yeah sorry this is a mistake; my IDE is eclipse which tries to guess where to import stuff from automatically. Doesn't always get it quite right :)